heksher/db_logic/setting.py

```diff
@@ -109,9 +129,15 @@ async def add_setting(self, setting: Setting):
                         [{'setting': setting.name, 'key': k, 'value': v} for (k, v) in setting.metadata.items()]
                     )
                 )
+            if alias:
+                await conn.execute(
+                    insert(setting_aliases).values(
+                        [{'setting': setting.name, 'alias': alias}]
+                    ).on_conflict_do_nothing()
```

Comment:
why? if the alias already exists for a different setting, you will get a conflict